packages/templates/cra-template-web-viewer/template/src/Auth.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { BrowserAuthorizationClientConfiguration } from "@itwin/browser-authorization";
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
+
+export class Auth {
+  private static client: BrowserAuthorizationClient;
+
+  public static initialize(
+    options: BrowserAuthorizationClientConfiguration
+  ): BrowserAuthorizationClient {
+    this.client = new BrowserAuthorizationClient(options);
+    return this.client;
+  }
+
+  public static getClient(): BrowserAuthorizationClient {
```

Comment:
```suggestion
  public static get client(): BrowserAuthorizationClient {
```

Reply:
Not really a fan of getters and setters. Underscore is fine.

Reply:
no worries, i had mentioned this to her. nbd

Reply:
Weird preferences, I'm probably in the minority here.